service/rds/api_op_CopyDBClusterSnapshot.go

```diff
@@ -385,27 +352,12 @@ func addCopyDBClusterSnapshotPresignURLMiddleware(stack *middleware.Stack, optio
 			CopyInput:            copyCopyDBClusterSnapshotInputForPresign,
 			SetDestinationRegion: setCopyDBClusterSnapshotdestinationRegion,
 			SetPresignedURL:      setCopyDBClusterSnapshotPreSignedUrl,
+			PresignOperation:     presignCopyDBClusterSnapshot,
```

Comment:
This doesn't really need to be a function using empty interfaces. If it were a method on a type that wrapped the presigned client, there is no need to pass the Presign client down via multiple levels of empty interfaces.